src/libs/ascent/ascent.hpp

```diff
@@ -66,13 +66,31 @@ class ASCENT_API Ascent
     conduit::Node  m_info;
 };
 
+// Callback maps
+static std::map<std::string, void (*)(conduit::Node &, conduit::Node &)> m_void_callback_map;
+static std::map<std::string, bool (*)(void)> m_bool_callback_map;
```

Comment:
we should define these in the cpp file, otherwise they could be multiple defined by users calling ascent.